src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/AuthenticationHelper.NtAuth.tvOS.cs

```diff
@@ -13,7 +13,7 @@ private static Task<HttpResponseMessage> InnerSendAsync(HttpRequestMessage reque
         {
             return isProxyAuth ?
                 SendWithProxyAuthAsync(request, authUri, async, credentials, true, connectionPool, cancellationToken).AsTask() :
```

Comment:
Since you are trying my "fix", this line should also be `false`.
```suggestion
                SendWithProxyAuthAsync(request, authUri, async, credentials, false, connectionPool, cancellationToken).AsTask() :
```

Reply:
Thanks :)